spec/models/company_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Company, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I assume this file will be useful soon, so it makes sense to keep it, but you could remove this line to get rid of the "pending" warning from your rspec's output.

```suggestion
```